firefox-ios/Storage/KeychainWrapperExtensions.swift

```diff
@@ -18,67 +18,3 @@ public extension MZKeychainWrapper {
         return MZKeychainWrapper(serviceName: baseBundleIdentifier, accessGroup: accessGroupIdentifier)
     }
 }
-
-public extension MZKeychainWrapper {
```

Comment:
I saw no evidence that the functions introduced in this extension were used. But let me know if that is incorrect so I can undo if necessary.